.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
-          # Combine existing content with new fix notes
-          updated_content="${existing_content}<br/><br/>${RELEASE_NOTES}"
+          # Remove '##' from RELEASE_NOTES
+          RELEASE_NOTES=$(echo -e "${RELEASE_NOTES}" | sed 's/## //g')
```

Comment:
- The final step of replacing '\n' with '<br/>' should be performed before combining the content, as it affects the structure of the text. Ensure the order of operations is logical and consistent.